README.md

```diff
@@ -14,7 +14,7 @@ You can find more details about prerequisites, instructions, and many useful tip
 ```bash
 $npx create-socket-app -h
 
-usage: create-socket-app [react | svelte | tonic | vanilla | vue]  
+usage: create-socket-app [react | react_ts | svelte | tonic | vanilla | vue]  
```

Comment:
Let's change `react_ts` to `react-ts`

I know this will cause an issue with the 1:1 templates[templateName] mapping, but it will have a nicer flow if it can be typed as 
```sh
npx create-socket-app react-ts`
```

You can modify index.js as follows:
```js
templates['react-ts'] = { //...
```
Rename react_js folder as react-js
And change any other references to `react_js` to `react-js`

Reply:
Done